### PR TITLE
DigiDNA March 2023

### PR DESCRIPTION
**Pull request by DigiYann**
* Updated DigiDNA December 2022 to DigiDNA March 2023 
* Updated com.apple.DirectoryService.managed.plist
* Updated com.apple.systemuiserver.plist
* Updated com.apple.finder.plist
* Updated com.apple.desktopservices.plist

### CODE REVIEW

**Comment by DigiYann** [comment]
* Desktopservices: Documentation link (Error 404) has been changed for another
* Nudge: PFC_JSON Preferences File added to SegmentedControl_0, pfm_app_min turned to 1.1.4, pfm_default to true, pfm_description text reworked, added a dictionary for JSON, integer updated to 2 (value)
* AssetCache.managed: pfm_title changed from Share Internet Connection to Disable Caching Over Shared Internet Connection
* DirectoryService.managed: Rework of pfm_conditionals on several keys
* Caldav.account: pfm_note changed
* Finder: pfm_macos_min added
* Loginitems.managed: pfm_documentation_url changed
* Loginwindow: pfm_description changed
* Proxy.http.global: pfm_description changed, pfm_require added, pfm_note chnaged
* Systemuiserver: pfm_description changed, Menu Extras value added to menu bar, dictionary (which was a double) merged from ManagedPreferencesApple to ManifestApple, pfm_title from Allowed Media to Allowed Media and Menu Bar Extras, integer from 4 to 5
* Webcontent-filter: pfm_default from true to false, pfm_note changed

File: Manifests/ManifestsApple/com.apple.caldav.account.plist [hunk 1]
```diff
@@ -246,7 +246,7 @@ In macOS, this key is optional.</string>
 			<key>pfm_name</key>
 			<string>CalDAVUseSSL</string>
 			<key>pfm_note</key>
-			<string>The official documentation as of writing shows the value of this property to be 'true', however in user reports and in further testing it was found to be the opposite case.</string>
+			<string>An issue resolved with the release of iOS 16 caused the actual default value of this preference on earlier versions of iOS to be false, as opposed to the documented value of true.</string>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.caldav.account.plist:249]: Please also update `pfm_last_modified`

File: Manifests/ManifestsApple/com.apple.systemuiserver.plist [hunk 2]
```diff
@@ -1559,17 +1564,70 @@
...
+			<string>List of menu extras to combine with the user's menu bar</string>
+			<key>pfm_name</key>
+			<string>menuExtras</string>
+			<key>pfm_title</key>
+			<string>Menu Extras</string>
+			<key>pfm_type</key>
+			<string>array</string>
+			<key>pfm_value_unique</key>
+			<true/>
+			<key>pfm_subkeys</key>
```
Comment (apizz) [inline Manifests/ManifestsApple/com.apple.systemuiserver.plist:1578]: Please sort this alphabetically in the list of keys at the same level, along with all key values within

**Review by apizz — CHANGES_REQUESTED** [review]
Almost there, just a few tweaks.  Thanks for this

**Pull request merged**